### ISSUE TITLE
Comments (or timestamp progress mechanism)

### ISSUE THREAD

**Issue by benjaminapetersen** [BaldissaraMatheus/Tasks.md#141]
Not sure what this would look like given each task is just a markdown file, but progressive updates on a task is handy when you can do something like:

`[+] add comment`

Then write the `comment` and have an automatic `timestamp`. I believe a number of Markdown implementations utilize a `Header` or `Front Matter` block, like `Yaml` that could be used for tracking right within the document:

```
---
title: Metadata stuff
author: me
things:
- hi
- mom
comments:
- timestamp: 2021-07-03 16:21:12.357246
   text: This is a comment
- timestamp:  2021-07-05 14:05:12.357432
   text: Yet another
---

# This Task

is a task I shall do
1. first
2. and then second
3. and of course third

```
Keeping a good minimalist approach, I would't expect threading or anything like that as `Tasks.md` seems well-suited for a self-managed task system running locally.  Having your own progress log on a task (auto-tracking timestamps, something like moment.js' `moment().format('MMMM Do YYYY, h:mm:ss a'); // December 6th 2024, 4:49:52 pm`) would be a really helpful feature.

**Comment by azzamsa** [comment]
Yeah, I also use Planka primary for this reason. I hope I can see this in Taks.md

**Comment by benjaminapetersen** [comment]
Planka looks nice if you want a similar UX and are good with data in a PostgresDB instead of .md files.

**Comment by BaldissaraMatheus** [comment]
I decided to not increase the scope of the project for now (as described in the "This is a low maintenance project segment" in README), so I'm closing this issue
